lib/twingly/http.rb

```diff
@@ -75,12 +77,12 @@ def initialize_defaults
       end
 
       # rubocop:disable Metrics/MethodLength
-      def http_response_for(method, *args)
+      def http_response_for(method:, **args)
         response = case method
                    when :get
-                     http_get_response(*args)
+                     http_get_response(**args)
                    when :post
-                     http_post_response(*args)
+                     http_post_response(**args)
```

Comment:
I only had to switch from `**` to `*` in order to get rid of the warnings, there seems to be no need for changing `method` into a keyword argument (`method:`).

I tested with the following changes, which didn't give me any warnings in 2.7.1:

```diff
diff --git a/lib/twingly/http.rb b/lib/twingly/http.rb
index 9ac7f9c..07b1629 100644
--- a/lib/twingly/http.rb
+++ b/lib/twingly/http.rb
@@ -52,13 +52,11 @@ module Twingly
       end

       def get(url, params: {}, headers: {})
-        http_response_for(method: :get, url: url,
-                          params: params, headers: headers)
+        http_response_for(:get, url: url, params: params, headers: headers)
       end

       def post(url, body:, headers: {})
-        http_response_for(method: :post, url: url,
-                          body: body, headers: headers)
+        http_response_for(:post, url: url, body: body, headers: headers)
       end

       private
@@ -77,7 +75,7 @@ module Twingly
       end

       # rubocop:disable Metrics/MethodLength
-      def http_response_for(method:, **args)
+      def http_response_for(method, **args)
         response = case method
                    when :get
                      http_get_response(**args)
```

Reply:
Yep you're correct, there was no need to change `method`... I don't remember how I reasoned there. 🤔